### PR TITLE
Remove --pre flag from ocean-lib installation. Update permissions for starting barge.

### LINKED ISSUE #1266: Inconsistent versions of Python used in ReadMe steps
**Describe the bug**
Inconsistent versions of Python used in install.md and setup-local.md. Change setup-local.md to use `python3` to match install.md's python3 venv setup.

**To Reproduce**
1. Go to setup-local.md section 4
2. See "In the same console, run Python console:" `python`

**Expected behavior**
Expected to be `python3`

**Additional context**
OS agnostic.

### PR DESCRIPTION
**Pull request by mariacarmina**
Closes #1266 #1268  .

Changes proposed in this PR:

- add permissions to `start_ocean.sh`
- remove `--pre` flag from ocean-lib installation
- update DDO samples

### CODE REVIEW

**Review by trentmc — APPROVED** [review]

**Comment by mariacarmina** [comment]
This issue should not be encountered when creating a python3 venv, because it automatically uses python3 and also the correct version of pip. Let's revert the changes and use everywhere in the READMEs python and pip.
cc @calina-c @trentmc

File: READMEs/c2d-flow-more-examples.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ ALGO_url_file = UrlFile(
 
 Install dependencies:
 
-`pip install Pillow`
+`pip3 install Pillow`
```
Comment (calina-c) [inline READMEs/c2d-flow-more-examples.md:79]: please revert changes referencing pip3 and python3. If the venv is activated, the correct version of python will be used.

File: READMEs/setup-local.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ cd barge
 # Clean up old containers (to be sure)
 docker system prune -a --volumes
 
+# Give permissions to the script
+sudo chmod +x start_ocean.sh
```
Comment (calina-c) [inline READMEs/setup-local.md:29]: please wrap this in a console or at least `code` markdown. I think the instruction above should be the same

File: ocean_lib/ocean/test/test_ocean_assets.py [hunk 3]
```diff
@@ -239,6 +239,7 @@ def test_ocean_assets_search(publisher_ocean, publisher_wallet):
     )
 
 
+@pytest.mark.skip(reason="Skip this until #1268 is merged.")
```
Comment (calina-c) [inline ocean_lib/ocean/test/test_ocean_assets.py:242]: @mariacarmina please fix inside this PR. All you need to do is cherry-pick 11ce25f09cf4873055740bf4eff17f49d3628671 and the subsequent changes here: https://github.com/oceanprotocol/ocean.py/pull/1265/files#diff-dfe25f756de0539701ed9f2643a7d93f22dfb4649eac3e4a6d7f0bd0820545c7

**Review by calina-c — CHANGES_REQUESTED** [review]

File: READMEs/developers.md [hunk 4]
```diff
@@ -36,7 +36,8 @@ cd ocean.py
 sudo apt-get install -y python3-dev gcc python-pytest
 
 #Initialize virtual environment and activate it.
-python -m venv venv
+#Use python3 to create the env for Python3.8, Python3.9, Pyhton3.10, Python3.11.
```
Comment (calina-c) [inline READMEs/developers.md:39]: there is a typo in Python3.10, also I think you can rephrase as: "make sure your Python version inside the venv is >=3.8."

Can you please add spaces after the `#` character throughout this file? It always bugs me and I don't want to perpetuate this

File: READMEs/setup-local.md (hunk 2 above)
Comment (calina-c) [reply READMEs/setup-local.md:29]: no, sorry. I missed the generic wrap :)

**Review by calina-c — COMMENTED** [review]
Just a small wording change, please.

**Review by calina-c — APPROVED** [review]

**Pull request merged**